cabal.project.freeze

```diff
@@ -395,7 +395,6 @@ constraints: any.Cabal ==3.10.3.0,
              any.some ==1.0.6,
              some +newtype-unsafe,
              any.sop-core ==0.5.0.2,
-             any.souffle-haskell ==4.0.0,
```

Comment:
Can you re-generate the file as a whole? This will also remove unneeded transitive dependencies.